app/test/unit/pull-request-database-test.ts

```diff
@@ -0,0 +1,38 @@
+import { expect } from 'chai'
+
+import {
+  PullRequestDatabase,
+  IPullRequestStatus,
+} from '../../src/lib/databases'
+
+describe('PullRequestDatabase', () => {
+  it("adds statuses key to records that don't have one on upgrade", async () => {
+    const databaseName = 'TestPullRequestDatabase'
+    let database = new PullRequestDatabase(databaseName, 3)
+
+    await database.delete()
+    await database.open()
+
+    const prStatus: IPullRequestStatus = {
+      id: 1,
+      pullRequestId: 1,
+      state: 'success',
+      totalCount: 4,
+      sha: 'sha',
+    }
+
+    await database.pullRequestStatus.add(prStatus)
+    await database.pullRequestStatus.each(prStatus => {
+      expect(prStatus.statuses).to.be.undefined
```

Comment:
Can we assert that a record exists in the database? This `expect` might not be asserted if the table is empty.